src/content/platform/languages.md

```diff
@@ -60,6 +60,7 @@ Since Workers runs off of JS, you can also program on Workers with any language
 | Python       | [cloudflare/python-worker-hello-world](https://github.com/cloudflare/python-worker-hello-world) |
 | Scala        | [cloudflare/scala-worker-hello-world](https://github.com/cloudflare/scala-worker-hello-world)   |
 | Reason/OCaml | [cloudflare/reason-worker-hello-world](https://github.com/cloudflare/reason-worker-hello-world) |
+| Perl         | [cloudflare/perl-worker-hello-world](https://github.com/cloudflare/perl-worker-hello-world) |
```

Comment:
Please line up the `|` at the end of the line with the table rows above it. It helps “visualize” the table in the markdown.